README.md

```diff
@@ -9,7 +9,7 @@ const {FullPage, Slide} = require('react-full-page');
 const FullPageExample = React.createClass({
   render() {
     return (
-      <FullPage>
+      <FullPage {/* initialSlide={1} */}>
```

Comment:
I don't think you really need curly brackets here

Reply:
I know, my intention was to show it's possible to add the prop but not mandatory, I'm happy to remove it.

Reply:
Oh, now I got it, sorry. I thought about this `<FullPage /* initialSlide={1} */>` but that is not a valid JSX.